spring-cloud-sleuth-zipkin/pom.xml

```diff
@@ -38,6 +38,10 @@
 			<artifactId>spring-cloud-sleuth-core</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-web</artifactId>
```

Comment:
Since this is a dependency to our Spring module that shouldn't  be a big problem